src/components/top-bar-navigation.svelte

```diff
@@ -0,0 +1,19 @@
+<nav class="navigation">
+    <a class="navi" href="carrer.html">carrers</a>
+    <a class="navi" href="help.html">help</a>
+    <a class="navi" href="buyer-protection.html">Buyer protection</a>
+</nav>
+
+<style>
+    .navi {
```

Comment:
Czy znalazłbyś może bardziej pasującą nazwę klasy?
Jako że wiemy że jesteśmy w nawigacji, a te elementy to są anchory - linki. Może klasa `link` byłaby lepszym wyborem?